src/simplifiers/LARewriter.cc

```diff
@@ -0,0 +1,111 @@
+//
+// Created by Martin Blicha on 03.02.21.
+//
+
+#include "LARewriter.h"
+
+#include "LIALogic.h"
+#include "OsmtApiException.h"
+
+std::size_t DivModRewriter::counter = 0;
+const char * DivModRewriter::divPrefix = ".div";
+const char * DivModRewriter::modPrefix = ".mod";
```

Comment:
I thought about if this should be in LIALogic or here in the rewriter. Now I am more inclined  to keep it here.
It feels closer to there it is actually used and I believe it should not cause any problems.